boards/shields/adafruit_24lc32/adafruit_24lc32.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 Jonas Berg
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		eeprom-0 = &eeprom_adafruit_24lc32;
+	};
+};
+
+&zephyr_i2c {
+	status = "okay";
+
+	eeprom_adafruit_24lc32: eeprom@50 {
+		status = "okay";
+		compatible = "atmel,at24";
```

Comment:
For demonstration purposes, could you please add a more precise compatible string as well? Similar as to what is done in https://github.com/zephyrproject-rtos/zephyr/blob/375d51452e80ed9fd2427492c41eb1163b0b8e49/boards/shields/x_nucleo_eeprma2/x_nucleo_eeprma2.overlay#L23

```suggestion
		compatible = "atmel,at24lc32", "atmel,at24";
```

Reply:
Done